migrations/000001_init_schema.down.sql

```diff
@@ -0,0 +1,2 @@
+-- +migrate Down
+DROP TABLE book;
```

Comment:
please add new line for end of file

Reply:
ok kang, but i have been curious is there a reason to add additional line in the of file?